Gemfile.lock

```diff
@@ -167,6 +170,9 @@ GEM
     dotenv-rails (3.1.0)
       dotenv (= 3.1.0)
       railties (>= 6.1)
+    double-bag-ftps (0.1.4)
```

Comment:
Q: will this be required when we ditch carrierwave for FTP?

Reply:
Said in another way: now that we are upgraded away from that carrierwave extra gem, does a bundle install remove this from the gemfile.lock?